pkg/apis/operator/v1alpha1/tektonpipeline_types.go

```diff
@@ -51,8 +51,8 @@ func (tp *TektonPipeline) GetStatus() TektonComponentStatus {
 
 // TektonPipelineSpec defines the desired state of TektonPipeline
 type TektonPipelineSpec struct {
-	CommonSpec         `json:",inline"`
-	PipelineProperties `json:",inline"`
+	CommonSpec `json:",inline"`
+	Pipeline   `json:",inline"`
```

Comment:
a structural suggestion. 

at present the type `Pipeline` is defined in `tektonconfig_types.go`. 

This part of the code will be more readable if we move the `Pipeline` struct into `tektonpipeline_types.go`, as we are using this struct inline in TektonPipelineSpec and in TektonConfigSpec as a named field. 

The main reason why this will be an improvement is because, github directed me to [here](https://github.com/tektoncd/operator/blob/27d571d2daa248856a8e53e361523a79fae10637/vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1/pipelinerun_types.go#L212:2). So it took deliberate look to find the right type definition. 🧑‍💻 It would have helped me a lot if the struct definition was in this file. 🙂